config/torch/optim/lr_scheduler.py

```diff
@@ -66,12 +67,12 @@ class CosineAnnealingLRConf:
 class ReduceLROnPlateauConf:
     _target_: str = "torch.optim.lr_scheduler.ReduceLROnPlateau"
     optimizer: Any = MISSING
-    mode: str = 'min'
+    mode: Any = min
```

Comment:
This breaks lr_scheduler. Those defaults have to be in quotes. I think this is just because when you ran configen, it didn't have the latest PR I made integrated.

Reply:
hmmmmm, strange, I have merged with latest hydra master today,

Anyway, let me manually revert that and we will deal with that later.

Reply:
done